Template/Tag/LivezillaDynamicTag.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+namespace Piwik\Plugins\TagManager\Template\Tag;
+
+use Piwik\Settings\FieldConfig;
+use Piwik\Plugins\TagManager\Template\Tag\BaseTag;
+use Piwik\Validators\CharacterLength;
+use Piwik\Validators\NotEmpty;
+use Piwik\Validators\NumberRange;
+use Piwik\Validators\UrlLike;
+
+class LivezillaDynamicTag extends BaseTag
+{
+	public function getIcon()
+    {
+        // You may optionally specify a path to an image icon URL, for example:
+        //
+        // return 'plugins/TagManager/images/MyIcon.png';
+        //
+        // to not return default icon call:
+        // return parent::getIcon();
+        //
+        // The image should have ideally a resolution of about 64x64 pixels.
+        return 'plugins/TagManager/images/icons/livezilla_icon.png';
+    }
+	
+	public function getCategory()
+    {
+        return self::CATEGORY_SOCIAL;
+    }
+
+    public function getParameters() {
+        return array(
+            $this->makeSetting('LivezillaDynamicID', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
+                $field->title = 'Livezilla ID';
+                $field->description = 'Insert the Livezilla ID from your Dynamic Code section.';
+                $field->validators[] = new NotEmpty();
+                $field->validators[] = new CharacterLength(32);
+            }),
+			$this->makeSetting('LivezillaDynamicDomain', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
+                $field->title = 'Livezilla Domain';
+                $field->description = 'Enter the Domain where Livezilla is installed. Example: https://www.example.com';
+                $field->validators[] = new NotEmpty();
+				$field->validators[] = new UrlLike();
+                $field->validators[] = new CharacterLength(11, 60);
+            }),
+			$this->makeSetting('LivezillaDynamicDefer', true, FieldConfig::TYPE_BOOL, function (FieldConfig $field) {
+                $field->title = 'Livezilla Script "defer"?';
+				$field->uiControl = FieldConfig::UI_CONTROL_CHECKBOX;
+                $field->description = 'In most cases you should let it activated.';
```

Comment:
If possible, be good to give an example when it makes sense to disable it. Not needed though.

Reply:
No its really not needed. But sometimes you have sites where you would remove this part. So there are 2 Options. Remove "defer" completely or leave it as default and give an user no option s to remove it.

Reply:
An really better solution would be an general option in Tag Manager to async or defer scripts. I don't find those options at this moment.

Example from Qubit´s OpenTag Manager
<img width="1156" alt="bildschirmfoto 2018-10-05 um 13 51 28" src="https://user-images.githubusercontent.com/1695531/46533701-de53e800-c8a5-11e8-9bff-d15f9a066477.png">